src/webapp/helpers/map-urgency.js

```diff
@@ -3,6 +3,9 @@
  * usable by Material UI.
  */
 export const urgencyStyles = {
+  invalid: {
+    backgroundColor: "black",
```

Comment:
i think the background color for this can be "grey" when this data is unavailable. this will downplay the importance of the "n/a" label.

Reply:
good point!